packages/fx-core/src/core/middleware/consolidateLocalRemote.ts

```diff
@@ -425,6 +420,20 @@ async function compareLocalAndRemoteManifest(
   }
 }
 
+async function notifyToUpdateManifest() {
+  const res = await TOOLS?.ui.showMessage(
+    "warn",
+    getLocalizedString("core.consolidateLocalRemote.DifferentManifest"),
+    false,
+    "OK",
+    LearnMore
+  );
+  const answer = res?.isOk() ? res.value : undefined;
+  if (answer === LearnMore) {
+    TOOLS?.ui.openUrl(LearnMoreLink);
```

Comment:
use different aka link, we'd better to direct users to one anchor of the wiki page.

Reply:
Fixed.